chapter_3/ex3_floating.rb

```diff
@@ -0,0 +1,33 @@
+# print a string
+puts 'I will now count my chickens:'
+
+# prints a combination of strings and code (used math arithmetic)
+puts 'Hens #{25.0+30.0/6.0}'
```

Comment:
If any of the numbers are floating point, it will switch to floating point math, so this statement is equivalent to `25 + 30 / 6.0}`.

Please switch to double quotes.